File: Luyencode/SCP - Đếm số chính phương/SCP.py
import math

def scp(x):
    return int(math.sqrt(x))

n = int(input())
for i in range (1, n + 1):
    m = int(input())
    if i != n:
        print(scp(m))
    else:
        print(scp(m), end="")

File: Luyencode/SCP - Đếm số chính phương/SCP.cpp
#include <bits/stdc++.h>
using namespace std;

long long SCP(long long n) {
    return (long long)sqrt(n);
}

int main() {
    long long n, m;
    cin >> n;
    for (int i = 1; i <= n; ++i) {
        cin >> m;
        if (i != n)
            cout << SCP(m) << endl;
        else
            cout << SCP(m);
    }
    return 0;
}